DashKitUI/Views/DashSettingsView.swift

```diff
@@ -19,7 +19,9 @@ struct DashSettingsView: View  {
     @State private var showingDeleteConfirmation = false
     
     @State private var showSuspendOptions = false;
-    
+
+    @State private var confidenceRemindersEnabled = false;
```

Comment:
[nit] Does this need to be persisted somewhere?

Reply:
It will be persisted in the ticket that actually implements the functionality; this is just making placeholder UI.